src/controls/TextControl.tsx

```diff
@@ -102,7 +100,12 @@ function TextControlInput({ type, ...rest }: TextControlInputProps) {
       // idk why type isn't getting narrowed properly here, but cast seems safe
       return <Input {...(rest as InputProps)} />
     case "password":
-      return <Input.Password {...(rest as InputProps)} />
+      // https://developer.mozilla.org/en-US/docs/Web/Security/Securing_your_site/Turning_off_form_autocompletion
+      return (
+        <Input.Password
+          {...({ ...rest, autoComplete: "new-password" } as InputProps)}
```

Comment:
Setting autoComplete to `"new-password"` will cause browsers to treat the field as a new password and offer to generate one, for instance. That may be a use case for some folks using this library, but I don't think it's safe to assume it's the primary use case. If you want it to have specific configurable behavior, I'd suggest modifying `TextControlOptions` in src/ui-schema.ts to have an `inputProps` property typed similarly to `TextControlInputProps` in this file, so that users can configured this behavior via a UISchema.

Reply:
it was just a workaround to prevent auto-filling the input cause as it's mentioned in the article I posted in the comment
> Even without a master password, in-browser password management is generally seen as a net gain for security. Since users do not have to remember passwords that the browser stores for them, they are able to choose stronger passwords than they would otherwise.
For this reason, many modern browsers do NOT support autocomplete="off" for login field

But they support `new-password` in the right way. 
I def can pass it as an input prop

Reply:
I think we should avoid accreting properties on TextControlOptions that are really just antd InputProps--did you try adding a `inputProps` property to TextControlOptions that's typed as antd's InputProps?

Reply:
yup, @DrewHoo can you take a look?